<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_OP1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_OP2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Операция</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Sqrt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Корень</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Sum">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Сложение</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Sub">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Вычитание</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Mul">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Умножение</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Div">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Деление</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Resault">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Resault">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_OP1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Операнд 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_OP2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Операнд 2</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>lineEdit_OP1</zorder>
   <zorder>lineEdit_OP2</zorder>
   <zorder>pushButton_Resault</zorder>
   <zorder>label_Resault</zorder>
   <zorder>label_OP1</zorder>
   <zorder>label_OP2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
